cores/arduino/main.cpp

```diff
@@ -54,10 +54,6 @@ int main( void )
 {
   initVariant();
```

Comment:
Question - Maple/STM32Generic have a USBInit call the variant can override if it needs to perform specific changes (oh, say, like turning on a pullup transistor that never should have been there in the first place). Would this be done normally in initVariant, or should there be a similar concept in this core to USBInit()?

Reply:
This is a good question. This will need I investigate this. Anyway, any feedback/comment are welcome.

Reply:
With #388 Updating PeripheralPins.c will allows to configure GPIO config.
`usbd_interface_init` is also a weaked function. (See #341)